src/modules/AddOnOthers.rb

```diff
@@ -0,0 +1,79 @@
+# encoding: utf-8
+require "yast"
+require "y2packager/resolvable"
+
+# Yast namespace
+module Yast
+  # This module provides integration of the add-on products
+
+  class AddOnOthersClass < Module
+           
+    include Yast::Logger
+
+    #     add_on_others = [
+    #        {
+    #          "media" => 4, # ID of the source
+    #          "name" : "openSUSE version XX.Y",
+    #          "media_url"=>"dvd:/?devices=/dev/sr1"
+    #          ....
+    #        },
+    #        ...
+    #      ]
+    attr_reader :add_on_others
+
+    def main
+      Yast.import "Pkg"
+      textdomain "packager"      
```

Comment:
It should be "add-on", to be consistent with the package name.

```suggestion
      textdomain "add-on"      
```